### PR TITLE
Include dependencies in jumpbox build

### PR DESCRIPTION
**Pull request by mjmenger**

### CODE REVIEW

File: dockerhost.userdata.tmpl [hunk 1]
```diff
@@ -8,4 +16,11 @@ sudo apt update
 sudo apt-add-repository --yes --update ppa:ansible/ansible
 sudo apt install software-properties-common ansible python-apt python-pip -y
 sudo pip install -q jmespath
-ansible-galaxy install -r ansible/requirements.yml
```
Comment (aknot242) [inline dockerhost.userdata.tmpl:11]: please add this line back. the ansible playbook will require this role

**Review by aknot242 — CHANGES_REQUESTED** [review]
What is the role of this jumphost?

File: dockerhost.userdata.tmpl (hunk 1 above)
Comment (mjmenger) [reply dockerhost.userdata.tmpl:11]: The jumphost is the box from which ansible jobs and similar are executed.

Comment (mjmenger) [reply dockerhost.userdata.tmpl:11]: and we can't add that line back because this file is executed before the requirements.yml file exists.

Comment (mjmenger) [reply dockerhost.userdata.tmpl:11]: do we still need to discuss this, or can we resolve the comment?

Comment (aknot242) [reply dockerhost.userdata.tmpl:11]: Yes, it can be resolved. Sorry 'bout that.

**Review by aknot242 — APPROVED** [review]

**Pull request merged**